packages/clerk-js/src/ui/components/AccountlessPrompt/ClerkLogoIcon.tsx

```diff
@@ -0,0 +1,73 @@
+// Custom icon specifically used for the Keyless toast. Not intended to be used anywhere else
+
+export function ClerkLogoIcon() {
+  return (
+    <svg
+      width='16'
+      height='20'
```

Comment:
```suggestion
      width='1rem'
      height='1.25rem'
```

Reply:
For reference, when using width and height as pixel values, they don't scale correctly with the rest of the UI.

![Screenshot 2024-12-04 at 9 12 20 AM](https://github.com/user-attachments/assets/2812cd52-85ff-4e64-83c5-24c0edc4a02a)